utility/utils.py

```diff
@@ -2798,3 +2799,93 @@ def is_unsecured_registry(test_data):
             return insecure_registry_value  # Return boolean value for insecure-registry
 
     return False  # Default return value if conditions are not met
+
+
+def lookup_in_directory(client, mount):
+    """
+    Check if the mount point is accessible and contains expected files.
+    Args:
+        client: Client node where the mount is performed.
+        mount: Mount point path.
+    Returns:
+        bool: True if the mount point is accessible and contains expected files, False otherwise.
+    """
+    try:
+        out, _ = client.exec_command(sudo=True, cmd=f"ls {mount}")
+        log.info(f"Contents of {mount}: {out.strip()}")
+        return out
+    except Exception as e:
+        log.error(f"Failed to access mount point {mount}: {e}")
+        return False
+
+
+def create_file(client, nfs_mount, file_name):
+    """Create a file in the NFS mount point"""
+    try:
+        if file_name not in lookup_in_directory(client, nfs_mount).strip().split():
+            cmd = f"touch {nfs_mount}/{file_name}"
+            client.exec_command(cmd=cmd, sudo=True)
+            log.info("File - {0} created successfully".format(file_name))
+    except Exception as e:
+        log.error(f"Failed to create file {file_name}: {e}")
+        raise OperationFailedError(f"Failed to create file {file_name}: {e}")
+
+
+def delete_file(client, nfs_mount, file_name):
+    """Delete a file in the NFS mount point"""
+    try:
+        cmd = f"rm -rf {nfs_mount}/{file_name}"
+        client.exec_command(cmd=cmd, sudo=True)
+        log.info("File - {0} deleted successfully".format(file_name))
+    except Exception as e:
+        log.error(f"Failed to delete file {file_name}: {e}")
+        raise OperationFailedError(f"Failed to delete file {file_name}: {e}")
+
+
+def rename_file(client, nfs_mount, old_name, new_name):
```

Comment:
We already have an existing method for rename.
Check - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/utils.py#L938